### DISCUSSION TITLE
Q about the extension

### DISCUSSION THREAD

**Discussion by mucaadciiltire** [sharafdin/markdown-master#5] [Q&A] [answered]
can i be contributor of the extension?

**Comment by sharafdin** [comment, marked as answer]
Absolutely! The extension is indeed open source, and you are more than welcome to become a contributor. We encourage individuals like you to join our project and contribute to its development. To get started, please visit our GitHub repository or contact our project manager to learn more about how you can participate and contribute your skills and expertise. Thank you for your interest in being a part of our open source project!

**Reply by mucaadciiltire** [reply]
thanks
